### ISSUE TITLE
Experimental reuslts on Human3.6M

### ISSUE THREAD

**Issue by MrZhangYH** [mks0601/3DMPPE_POSENET_RELEASE#1]
Hi Gyeongsik,

Good work.
In your paper, the network from "Integral human estimation regression (Sun)" is used as your PoseNet. However, the MPJPE achieved by your PoseNet is less than Sun (6.6mm). Why is this happening?

**Comment by mks0601** [comment]
Hi, that thing is related with detailed things. 

In the training stage, they compute bounding box using groundtruth 3D root coordinates. Specifically, they draw a fixed-size cubic around the human root coordinate and project it to 2D space to get 2D bounding box. And the same thing is applied in the testing stage. 

I tried that approach and result is really close to the Sun etal. However, that approach requires groundtruth 3D root coordinates. I can use RootNet estimation, but it when there are many people such as in the wild images, the accuracy of the RootNet becomes worse. That results in bad bounding box and bad 3D pose accuracy.

Instead, I just used bounding box from DetectNet. Although that gives me a little bit bad performance on H36M dataset, this approach gives better performance for the in-the-wild images.
